HW1/P2/.ipynb_checkpoints/205_2-checkpoint.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
In future, just for ease of readability, could you potentially gitignore the iPython Notebook stuff? Unless you actually intend for this to be looked at - it's kind of hard to comment on a .ipynb file though